### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by Frassle**
Based on one from ArasP:
http://aras-p.info/blog/2017/05/07/Users-POV-and-Empathy/

### CODE REVIEW

**Comment by Frassle** [comment]
Yes it's optional. It's just a suggested layout for pull requests like we have one for issues.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Purpose of this PR
+
+* Description of feature/change.
+* Links to screenshots, design docs, user docs, etc.
```
Comment (varon) [inline .github/PULL_REQUEST_TEMPLATE.md]: We definitely want to mention that they should include related issues here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* Explanation of what’s tested, how tested and existing or new automation tests.
+* Can include manual testing by self.
+* Specify test plans.
+* Rarely acceptable to have no testing.
+
+### Technical risk
+
+* Overall product level assessment of risk of change. 
+* Need technical risk & halo effect.
```
Comment (varon) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would drop this section, typically PRs are related to issues, which are discussed before.

We're not really using the WIP PR or speculative PR styles.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (varon) [reply .github/PULL_REQUEST_TEMPLATE.md]: We also could add which part of OpenTK this affects (Math, Core, GL, platform, etc)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### Comments to reviewers
+
+* Info per person for what to focus on, or historical info to understand who have previously reviewed and coverage. Help them get context.
```
Comment (varon) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would just drop this section or just generalize it to "Comments"

**Comment by varon** [comment]
Thanks!

**Pull request merged**